sellercenter-setting/src/main/java/com/gfg/sellercenter/setting/reader/HttpReader.java

```diff
@@ -0,0 +1,65 @@
+package com.gfg.sellercenter.setting.reader;
+
+import java.net.URL;
+import org.json.JSONObject;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import lombok.AllArgsConstructor;
+import java.nio.charset.StandardCharsets;
+
+@AllArgsConstructor
+public class HttpReader
+{
+    private String fqdnUrl;
+
+    private String query;
+
+    public JSONObject getSetting(String label, int sellerId) throws Exception
+    {
+        return createJsonObject(
+            loadResponse("?field=" + label + "&sellerId=" + sellerId)
+        );
+    }
+
+    public JSONObject getSetting(String label) throws Exception
+    {
+        return createJsonObject(
+            loadResponse("?field=" + label)
+        );
+    }
+
+    private JSONObject createJsonObject(String response)
+    {
+        if (response == null) {
```

Comment:
My opinion that it is better to not hide that we had a problem and fail as soon as possible. As client of this service I would expect following behaviour: always return Setting or throw an exception.